router/pump.go

```diff
@@ -60,10 +60,9 @@ func ignoreContainer(container *docker.Container) bool {
 			return true
 		}
 	}
-	excludeLabel := getopt("EXCLUDE", "")
-	_, ok := container.Config.Labels[excludeLabel]
-	if ok {
-		return true
+	excludeLabel := getopt("EXCLUDE_LABEL", "")
+	if value, ok := container.Config.Labels[excludeLabel]; ok {
```

Comment:
If `EXCLUDE_LABEL` is an empty string then containers with a label that has an empty key get ignored:

`$ docker run -d --label =true image`